vm/chef/cookbooks/django/files/django-config-setup

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash -eu
+#
+# Copyright 2021 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source /opt/c2d/c2d-utils || exit 1
+
+readonly default_site="$(get_attribute_value "default_site_name")"
+readonly external_ip="$(get_external_ip)"
+
+# Get external IP address.
+if has_external_ip; then
+  args+=(--url="$(get_external_ip)")
+else
+  args+=(--url="$(get_internal_ip)")
+fi
+
+mkdir /opt/django/
+cd /opt/django/
+
+# Create default project.
+django-admin startproject ${default_site}
+
+# Check if a directory exists.
+if [ -d "/opt/django/${default_site}" ] 
+ then
+    echo "Directory ${default_site} exists." 
+ else
+    echo "Error: Directory ${default_site} does not exists."
+fi
+
+# Whitelist external IP address.
+sed -i -e "s@ALLOWED_HOSTS = \[]@ALLOWED_HOSTS = ['${external_ip}']@g" /opt/django/${default_site}/${default_site}/settings.py
```

Comment:
enclose the settings.py path in `" "`

Reply:
imo makes sense to run this command only `if has_external_ip`